web/src/main/java/com/epam/rft/atsy/web/controllers/ApplicationStateController.java

```diff
@@ -43,6 +46,12 @@
   private StateFlowService stateFlowService;
 
   @Resource
+  private ApplicationsService applicationsService;
```

Comment:
This dependency is no longer referenced in the controller methods, it can be removed.